ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClustersCoverageTable.tsx

```diff
@@ -72,7 +70,7 @@ function ClustersCoverageTable() {
                 <Tr key={cluster.clusterId}>
                     <Td dataLabel="Cluster">
                         <Link
-                            to={generatePath(complianceEnhancedCoverageClustersPath, {
+                            to={generatePath(clusterDetailsRoute, {
```

Comment:
An observation, not necessarily a change request: my eyes stopped to look twice at the contrast
* name from within container `clusterDetailsRoute`
* name from outside container `complianceEnhancedSchedulesPath`

Reply:
I think with "complianceEnhanced" being so long, I've mistakenly shy'd away from using it in certain instances.